bundle/filefs_test.go

```diff
@@ -19,10 +19,7 @@ func TestFSLoader(t *testing.T) {
 		}
 	}
 
-	loader, err := NewFSLoader(archiveFS)
-	if err != nil {
-		t.Fatalf("Unexpected error: %s", err)
-	}
+	loader, _ := NewFSLoader(archiveFS)
 
```

Comment:
Nit: We don't need to change this as in the future it's possible we change the implementation of `NewFSLoader` such that it may return an error.

Reply:
Now reverting to check for error